### ISSUE TITLE
Feature Request - Automatic Annotation - Segment Anything Model - Run on task

### ISSUE THREAD

**Issue by weertman** [opencv/cvat#6217]
Feature request, wasn't sure where else to post.

The Segment Anything Model is fantastic! But it is a little slow. 

1. It would be really nice to have the option to run it on tasks prior to opening the them so that the interactions with the models output do not require initially waiting for each request on each image. 
2. It would be nice to be able to upload bounding box annotations and that have those annotated by Segment Anything Model. 

Thank you! Sorry, if this was posted elsewhere.

**Comment by priyanshum-cashify** [comment]
Yes, this feature will definitely help us in Running automatic annotation using SAM, will save lots of time.
**Upvote**

**Comment by saadehmd** [comment]
Has there been any development on this. Or is there any other way around this i.e.; increasing shm-size etc.

**Comment by DModjeski** [comment]
Following up on this: Is there any work going towards this issue? Having a solution to this would greatly cut down on the time to label per image.

I could see this being solved two different ways. 
1. Some sort of selection on the project (or Task) to state that by default SAM should run on all images
2. Allow for the upload of SAM returns on a data set similar to uploading a dataset that already has masks but hide all masks and then load them in the same way the interactor does.

**Comment by KDeser** [comment]
This would make labeling massively faster, especially #1
